docs/evm_compatibility/overview.md

```diff
@@ -40,24 +40,39 @@ Interoperability between Solana and Ethereum EVMs does require certain adaptatio
 ### Precompiles
 Neon supports all precompiled contracts defined on [evm.code](https://www.evm.codes/precompiled?fork=merge) that provide more advanced functionalities, with [certain limitations](./precompiles#limitations) on some precompiled contracts on Neon EVM.
 
+
+Neon also supports native precompiled contracts that are available to our users.
+
+
 ### Gas calculation
-The mechanism of gas consumption and calculation of gas fees on Neon EVM differ from Ethereum. Gas fees on Neon EVM are much cheaper than on Ethereum. 
+The mechanism of gas consumption and calculation of gas fees on Neon EVM differ from Ethereum. Gas fees on Neon EVM are much cheaper than on Ethereum because Solana is the settlement layer. 
+
+<!-- Oleg could provide metrics on one transfer. Once mainnet launched can we compare Neon + Solana to L2 and Rollups Yuri has for tx such as transfers and swaps -->
 
 > Learn more about the [NEON token and how gas fees work on Neon EVM](../../docs/tokens/gas_fees.md).
 
 Several Solana-specific differences also impact smart contract development.
 
-### Upper limit on number of Accounts
-Neon EVM uses Solana Transaction V0 with an extension via [account lookup tables](https://docs.solana.com/developing/lookup-tables): this limits the maximum number of accounts to 64 in a single transaction. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions. 
+### Upper limit on number of accounts
+<!-- link to solana tx renamed and relinked -->
+Neon EVM uses [Solana Transaction V0](https://docs.solana.com/developing/versioned-transactions): limiting the maximum number of accounts used in a single transaction to 64. Solana requires that all accounts used in a transaction be specified in order; to ensure parallel execution of transactions.
+
+<!-- go deeper on HOW to modify the contract to constrain account numbers Anton will pass in slack
+  -->
 
 ### Heap size
 Ethereum-like transactions are executed by Neon EVM inside [Solana's Berkeley Packet Filter (BPF)](https://docs.solana.com/developing/on-chain-programs/overview#berkeley-packet-filter-bpf). The BPF has heap memory limit of 256 KB. Consequently, the size of the heap allocated to a contract call, is limited to the same 256 KB.
 
-To avoid the occurrence of a heap overflow error, it is recommended that you:
-* Reduce the size of the call stack
-* Reduce the number of variables used
+To avoid the occurrence of a heap overflow error, it's recommended that you reduce the size of the:
+- call stack
+- contract
 
-### Limitation on `block.timestamp` / `block.number` Usage
+<!-- support users further on HOW to reduce heap size?
+
```

Comment:
@gigimon We have a note that you can provide information on troubleshooting to id when heap size is the cause of issues

Reply:
I can't describe this section :(

Reply:
@anton-lisanin was it you who could provide info on the heap size troubleshooting?